screens/UserListScreen.tsx

```diff
@@ -62,11 +83,16 @@ export const UserListScreen = () => {
             flexDirection: "row",
           }}
         >
-          <Image
-            source={{ uri: item.thumbnail }}
-            style={{ width: 40, height: 40 }}
-          />
-          <Text style={styles.item}>{item.name}</Text>
+          <TouchableOpacity
+            style={styles.item}
+            onPress={() => onPressUserDetails(item)}
```

Comment:
[non-blocking]
これでも大丈夫そう
```suggestion
            onPress={onPressUserDetails}
```

Reply:
こちら上記のように変更すると、詳細ページにデータが反映されなくなってしまったのですが、通常はうまくいくのでしょうか？
いったんこのままにしようと思います😳

Reply:
あら、そうでしたか。適当なこと言ってすみません:pray: